src/Maestro/Client/src/Generated/Models/AssetData.cs

```diff
@@ -22,5 +22,8 @@ public AssetData(bool nonShipping)
 
         [JsonProperty("locations")]
         public IImmutableList<Models.AssetLocationData> Locations { get; set; }
+
+        [JsonProperty("category")]
+        public string Category { get; set; }
```

Comment:
We can't modify the AssetData here. When we regenerate the model, it will remove this property

Reply:
So basically I think we need to simply not use AssetData as the intermediate type that is used to construct the merged manifest.

Reply:
I hate me :( I will fix this..

Reply:
I fixed this